src/main.ts

```diff
@@ -91,6 +92,8 @@ async function findAllRepos(
       username
     })
 
+    //console.log(repos)
+
```

Comment:
Let's not add extra empty lines, please remove

Reply:
removed empty lines